scripts/examples/index.csv

```diff
@@ -1,6 +1,7 @@
 # "Relative Path Regex", "Board Type Regex", "Sensor Type Regex", "Flatten Regex"
 "examples/00-HelloWorld/blinky.py", ".+", ".+", ""
 "examples/00-HelloWorld/helloworld.py", ".+", "^(?!None).*$", ""
+"examples/00-HelloWorld/pure_thermal_main.py", "(OPENMVPT)", "^(?!None).*$", ""
```

Comment:
I'm not so sure about this, helloworld examples are supposed to be really generic, basic stuff. Maybe you can add a new examples directory for PT in `50-OpenMV-Boards/PureThermal` ?

Reply:
Sure, will do.